contrib/cirrus/setup_environment.sh

```diff
@@ -61,7 +79,7 @@ case "$CG_FS_TYPE" in
             # This is necessary since we've built/installed from source,
             # which uses runc as the default.
             warn "Forcing testing with crun instead of runc"
-            echo "export OCI_RUNTIME=crun" >> /etc/environment
+            echo "OCI_RUNTIME=crun" >> /etc/ci_environment
```

Comment:
I really dislike how in some places (above this line) you just `echo >>/etc/ci_environment`, and in other places (below this line) you both set the envariable and then `echo` it. Aside from DRY, it makes me wonder why some envariables need that treatment and some don't. Could you just make them all a single `echo >>`, then `source /etc/ci_environment` at the end?

Reply:
Oh, I didn't notice the extraneous echo's, they must be left over from previous implementation...hmmm,  actually, I think you're referring to the `export` then `echo`.  Okay, simplifying those...